src/ui/svg/icons/types.ts

```diff
@@ -6,6 +6,13 @@ export interface IconInterface {
   testID?: string
 }
 
+export interface RectangleIconInterface {
```

Comment:
```suggestion
export interface RectangleIcon {
```
Même commentaire que l'enum du coup^^